qutebrowser-evil.el

```diff
@@ -32,17 +32,19 @@
     ("KeyMode.caret" . evil-visual-char)
     ("KeyMode.hint" . evil-motion-state)
     ("KeyMode.command" . evil-emacs-state)
-    ("KeyMode.normal" . evil-normal-state)))
+    ("KeyMode.normal" . evil-normal-state)
+    ("KeyMode.passthrough" . evil-emacs-state)))
 
 (defun qutebrowser-evil-update-state (window-info)
   "Set evil state to match Qutebrowser keymode from WINDOW-INFO."
   (when-let* ((x11-win-id (plist-get window-info :x11-win-id))
               (buffer (exwm--id->buffer x11-win-id)))
     (with-current-buffer buffer
       (qutebrowser--with-plist-key mode window-info
-        (let ((func (alist-get mode qutebrowser-evil-state-mappings
-                               nil nil 'string-equal)))
-          (funcall func))))))
+        (if-let ((func (alist-get mode qutebrowser-evil-state-mappings
+				  nil nil 'string-equal)))
+	    (funcall func)
+	  (evil-operator-state))))))
```

Comment:
Instead of using an if-let here, we can supply the default value in the call to `alist-get`.

```suggestion
        (let ((func (alist-get mode qutebrowser-evil-state-mappings
                               'evil-operator-state nil 'string-equal)))
          (funcall func))))))
```